README.md

```diff
@@ -141,14 +141,14 @@ Gogo support has the following limitations:
   // x must be greater than or equal to 20
   uint64 x = 1 [(validate.rules).uint64.gte = 20];
 
-  // x must be in the range [30, 40)
+  // x must be in the range [30, 40]
```

Comment:
```
// x must be in between 30 (inclusive) and 40 (exclusive)
```

Reply:
or maybe the more mathematically familiar: `30 <= x < 40`

Reply:
I thought it might have some meaning -- probably could have done a little searching :) this distinction (`30 <= x < 40`) is definitely more recognizable (to me, and presumably others..) Lmk if you'd like the change and I'll update, otherwise feel free to close this out!

Reply:
It'd be great if you could do it! Otherwise, if you can convert this PR to an issue, I can get updating the docs in the future 😁

Reply:
you bet, will get on it today 👍